TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Command/NodeCommandControllerPlugin.php

```diff
@@ -1005,9 +1005,11 @@ protected function collectNodesWithInvalidDimensions($workspaceName, array $allo
             foreach ($allowedDimensionCombinations as $dimensionConfiguration) {
                 ksort($dimensionConfiguration);
                 ksort($nodeDataArray['dimensionValues']);
-                if ($dimensionConfiguration === $nodeDataArray['dimensionValues']) {
-                    $foundValidDimensionValues = true;
-                    break;
+                foreach ($dimensionConfiguration as $dimensionKey => $dimensionValuesArray) {
+                    if (in_array($nodeDataArray['dimensionValues'][$dimensionKey][0], $dimensionValuesArray)) {
```

Comment:
Do I need to check if this value this key exists beforehand `$nodeDataArray['dimensionValues'][$dimensionKey][0]`? It doesn't seem to give any errors if it doesn't...